projects/dsp-ui/src/lib/action/components/message/message.component.html

```diff
@@ -25,11 +31,24 @@
 
     </mat-card-content>
 
-    <mat-card-footer *ngIf="!medium" class="message-footnote" [innerHtml]="message?.footnote"></mat-card-footer>
+    <mat-card-footer *ngIf="(size === 'long')" class="message-footnote">
+        <!-- Specific message from @Input -->
+        <p *ngIf="message.footnote">{{message.footnote}}</p>
+
+        <!-- Else: Default message -->
+        <p *ngIf="!message.footnote && !showLinks">
+            Please come back in a few minutes and try to <a class="link bolder" (click)="reload()">reload the page</a>.
+        </p>
+
+        <!-- Action: Contact DSP support -->
+        <a mat-button class="action" href="https://docs.dasch.swiss/community/faq" target="_blank">
+            <mat-icon>mail_outline</mat-icon> Contact DSP support
```

Comment:
DSP or DaSCH? Yesterday we used DaSCH https://github.com/dasch-swiss/dsp-ui-lib/pull/200

Reply:
DaSCH
I will update

Reply:
Updated (e10af34) with "Contact the support team" as mentioned in this comment:
<https://dasch.myjetbrains.com/youtrack/issue/DSP-710#focus=Comments-4-1211.0-0>